### ISSUE TITLE
Sentry helm chart + LDAP

### ISSUE THREAD

**Issue by sergiochoo** [sentry-kubernetes/charts#311]
Hello.
Is it possible to configure LDAP authentication with this helm chart (Sentry 21.1.0)? The official site has [documentation](https://docs.sentry.io/product/accounts/sso/azure-sso/) on configuring Azure AD, but not on configuring on-premise AD.
If it is possible, maybe there is a link to the instructions?
Thanks!

**Comment by nilpntr** [comment]
Yoo, you can mount a custom sentry.conf.py file and then use the Django ldap stuff. You have to install this plugin for python first and then use the configuration described over there: https://pypi.org/project/sentry-ldap-auth/

**Comment by Mokto** [comment]
This issue is stale because it has been open for 30 days with no activity.

**Comment by Mokto** [comment]
This issue was closed because it has been inactive for 14 days since being marked as stale.

**Comment by bumarcell** [comment]
@nilpntr is `sentry.conf.py` being used in the chart?
I searched this repo and found nothing. In the official repo this script is only used with tests..

Context is I'm trying to set up external kafka broker with ssl auth. I added the properties for that as following:
![image](https://user-images.githubusercontent.com/28063982/181531556-02390513-3f81-4135-998a-a0678a6c01ef.png)
and mounted the files as well. The ingest-consumer for example is still failing at kafka auth:
![image](https://user-images.githubusercontent.com/28063982/181532824-ba5e6895-29df-4608-86da-283957f9beaa.png)

Do I have to add that in [`server.py`](https://github.com/getsentry/sentry/blob/master/src/sentry/conf/server.py#L2350-L2359) and rebuild sentry myself?
